.gitignore

```diff
@@ -2,3 +2,4 @@ target
 /build
 .idea
 .venv
+.vs
```

Comment:
FYI, the commit message changing this mentioned VS Code, but that uses `.vscode` instead.

Reply:
Argh thanks! Fixed up.

As a Windows-outsider it boggles the mind that there are two, entirely separate, IDEs both practically named "Visual Studio" but they aren't replacements for one another!

For a brief period I was going crazy trying to figure out how to use `dumpbin` from a powershell prompt in VS Code until I realized I had to use Visual Studio (_not Code_) to launch a special developer powershell that had its path preconfigured for a bunch of useful dev tools. :dizzy_face: